.github/workflows/tests.yml

```diff
@@ -3,17 +3,16 @@ name: Tests
 on:
   pull_request:
   push:
-    branches: [main]
+    branches: [main, build_slim_gh_Actions]
```

Comment:
we should probably remove this other branch push action... was using this to develop

Reply:
ah, yep. should do.

Reply:
done